### PR TITLE
Final

### PR DESCRIPTION
**Pull request by 2bleO**
In this PR we are:

- Merging final project to main branch
- Updating Readme file to include project info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,79 @@
...
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+[Space-travelers-hub](https://vigorous-jang-d9676c.netlify.app)
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Project Overview Video
 
-### `npm test`
+[Watch Here](https://www.loom.com/share/22b0607f2e7348cb9575c43d241d600c)
```
Comment (vichuge) [inline README.md]: - [ ] Talking about the video, I consider it important to make the video again, now follow these requirements when you work on this: 🐶 
- All the members should appear in the video and talk about the project.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [x] Please, add tests, on this project you don't have them. 🐱 

- [x] Check the comments under the review.
# (It looks like the pull request doesn't have the component `Rockets.js` so, I'm gonna add this changes here)
- [x] When you use the dispatch to change the rocket status, you send all information, please change this to use the actions considering only the rocket id 
- [ ] At same time, it looks like you're not updating the local props once you press the button, please change the local props using useState. This change is for Missions.js too.
- [x] It's ok if you use one or two disables on linters to continue, but on this part I consider you use more than accepter disables for linters, please, fix this inside your script in place to disable a lot of rules. 🐱 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (2bleO) [reply README.md]: Hi  @vichuge 
May you please be a bit more specific with the following request?:

> * [ ]  At same time, it looks like you're not updating the local props once you press the button, please change the local props using useState. This change is for Missions.js too.

I am using this to get the state:
`const missions = useSelector((state) => state.missionReducer);`

and when Button is clicked an action is dispatched to update the state:
`<Button variant="outline-danger" id={mission.mission_id} onClick={leave}>Leave&nbsp;Mission</Button>`

**Review by ZawadiSumba66 — APPROVED** [review]
## Hi Team, 
## STATUS APPROVED ✔️ 
Great job on the remarkable effort on this project. 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**